### ISSUE TITLE
Release 1.4.1

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-core#737]
- [x] Release is scheduled and a ticket to track the release process is created.
- [x] Tag release on webmaker-core repo.
- [x] Generate signed APK.
- [x] Release notes are drafted in the ticket.
- [x] Ticket is assigned to deploy APK to beta channel.
- [x] Once APK has been released to beta channel wait about 4 hours.
- [x] QA beta on several devices (flag people to test in release ticket) (Set tag: Needs QA).
- [x] Once everyone flagged for QA approves the release, change tag to Passed QA.
- [ ] Check crash logs to ensure that no significant problems were reported.
- [ ] Promote beta APK to production.
- [ ] Send out email with release notes.
- [ ] File issues for follow up research.
- [ ] Close the release ticket.

**Comment by xmatthewx** [comment]
blocked by 
- [x] #711 FTU
- [x] #715 Botton text color

**Comment by ioana-chiorean** [comment]
hey @xmatthewx - I can help with " QA beta on several devices (flag people to test in release ticket) (Set tag: Needs QA)." I have at access several devices ( more than 30 - some old, some news etc)

**Comment by xmatthewx** [comment]
Thanks @ioana-chiorean - that's fantastic. 

The easiest way to participate: 
- pick one of these to test (Editing and Elements would be fun and useful): https://github.com/mozilla/webmaker-core/wiki/Webmaker-QA-Template-Issues
- open the sample issue. copy the text. create a new issue, paste and save it. 
- you can now use the checkboxes to guide your test

Find me or @vazquez on irc as mattheww and ricardo if you have any questions.

**Comment by xmatthewx** [comment]
@gvn - QA is complete. You can continue with the fix and the release checklist above.

**Comment by xmatthewx** [comment]
Release notes for future reference: 

> Webmaker has been updated again!
> - Load more projects in Discover.
> - Use round corners to crop images into circles.
> - Get help with the tap of a button.
> - Add text more quickly to your pages.
> - Use large italic text.
> - Enjoy various bug fixes and improvements.
> 
> The app is now translated to Bengali (India) thanks to our community!

**Comment by gvn** [comment]
This is actually 1.4.1 since it's just a hotfix patch. 1.5.0 is still "under construction".

**Comment by xmatthewx** [comment]
right-o

**Comment by gvn** [comment]
1.4.1 APKs are now uploaded into the beta channel. I think it takes a few hours for them to propagate though...

**Comment by xmatthewx** [comment]
Excellent. I'm having some nexus charging issues but will try this out today. 

@vazquez @flukeout  - take this for a spin as well. let's make sure this list of promises is true: https://github.com/mozilla/webmaker-core/issues/737#issuecomment-153143099. if so, we can push it to production.

**Comment by xmatthewx** [comment]
@vazquez @flukeout - you can confirm you're on 1.4.1 instead of 1.4 via Android Settings > Apps > Webmaker

**Comment by xmatthewx** [comment]
@gvn - it looks like 1.4.1 is hooked up with staging user accounts.

**Comment by gvn** [comment]
Ah crap, you're right. I forgot to switch the .env over. I'll put a new one up...

**Comment by gvn** [comment]
Ok, new APK is up and connected to prod servers. Just give it a few hours to propagate...

**Comment by xmatthewx** [comment]
New APK didn't appear for me in the play store. I had to delete and then install. Not ideal, but won't affect anyone who didn't opt into beta. Odd. 

I don't seem to have the new text color options for the link element. Just background color. Confirmed that I have 1.4.1 installed.

**Comment by ioana-chiorean** [comment]
@xmatthewx for - I don't seem to have the new text color options for the link element. Just background color. Confirmed that I have 1.4.1 installed.  

I commented yesterday - did not see it either

**Comment by gvn** [comment]
I haven't promoted the APK to production. It's still in the beta channel. Were you installing from the link or the play store?

**Comment by xmatthewx** [comment]
1.4.1 is definitely from Beta. Once you opt-in, you continue to get beta unless you choose to exit.

**Comment by gvn** [comment]
The commit I cherry picked as a patch to resolve #715 to unblock our release is [this one](https://github.com/mozilla/webmaker-core/commit/398ffb5464d1c84dc8bb1190f2260f4e68ebbfb3).

@xmatthewx are you referring to the functionality of [this patch](https://github.com/mozilla/webmaker-core/commit/2236f99832a554e33502b0a2e2f1b0983e038aab) being missing? That one isn't part of the 1.4.1 release...

Let me know when we're ready to promote 1.4.1 to prod and I can make it so!

**Comment by xmatthewx** [comment]
Took a quick look at this with @alanmoo. It seems like we might have rolled back a few commits that were initially in beta 1.4. I believe we had everything thru Oct 7, 2015 up to 0a621d5. The cherry picked patch fixes link color that landed on that date. So the fix doesn't have anything to fix.  https://github.com/mozilla/webmaker-core/pull/696

Fortunately, right now nothing is broken. So, we can choose to: 
- Rebuild 1.4.1 and push to production
- Push 1.4.1 to production & begin work on 1.5 Beta

Regardless, we should get something to production by noon-ish your time so we can capture new downloads from Mozfest people. We'll need to ramp up the release to 100%. I think it's low risk.

**Comment by gvn** [comment]
I've promoted [1.4.1](https://github.com/mozilla/webmaker-core/releases/tag/v1.4.1) to prod as is. I think some things made it into the last beta that weren't actually merged into master. Now that we're starting to tag releases things should become clearer moving forward...

**Comment by xmatthewx** [comment]
done and done
